docs/quick_start/north.rst

```diff
@@ -43,8 +51,18 @@ If Fledge is unable to send data to another system via a north service it will w
 
 The failure could be an incorrect configuration within Fledge for the particular north plugin or it may be caused by the upstream system being unavailable or there being no network route to the upstream system.
 
+Once the failure is cleared, the alert will be removed from the status bar.
+
+.. note::
+
+   If Fledge is shutdown and when it is later restarted the problem has cleared, the alert may persist in the status bar and should be manually cleared. If the alert is manually cleared but the problem persists then the alert will be recreated.
+
 Using the OMF plugin
 ####################
 
-OSISoft data historians are one of the most common destinations for Fledge data.  Fledge supports the full range of OSISoft historians; the PI System, Edge Data Store (EDS) and OSISoft Cloud Services (OCS). To send data to a PI server you may use either the older PI Connector Relay or the newer PI Web API OMF endpoint. It is recommended that new users use the PI Web API OMF endpoint rather then the Connector Relay which is no longer supported by OSIsoft.
+AVEVA PI (formally OSISoft PI) data historians are one of the most common destinations for Fledge data.  Fledge supports the full range of AVEVA historians; the PI System, Edge Data Store (EDS), OSISoft Cloud Services (OCS) and AVEVA Data Hub (ADH). To send data to a PI server you may use either the older PI Connector Relay or the newer PI Web API OMF endpoint. It is recommended that new users use the PI Web API OMF endpoint rather then the Connector Relay which is no longer supported by AVEVA.
```

Comment:
- formally -> formerly
- After "full range of AVEVA historians," use a colon rather than a semicolon.
- OSISoft Cloud Services -> OSIsoft Cloud Services
- To send data to a PI server -> To send data to a PI Server
- endpoint rather **then** -> endpoint rather **than**